dev/nfs/Makefile

```diff
@@ -0,0 +1,211 @@
+
+ifeq ($(VERDIN_MAC), CHANGEME)
+  $(error Please set the verdins MAC address in ot3-bootserver.env)
+else ifeq ($(DHCP_SUBNET_STUB), CHANGEME)
+  $(error Please set the DHCP subnet stub in ot3-bootserver.env)
+else ifeq ($(IFACE), CHANGEME)
+  $(error Please set your iface in ot3-bootserver.env)
+endif
+
+define noroutemsg
+
+There is no entry in the route table for the selected DHCP subnet stub
+$(DHCP_SUBNET_STUB).0. This either means your board is not plugged in
+to an ethernet interface, or you have not yet set the iface for the board
+to the static ip $(DHCP_SUBNET_STUB).1. This will not break anything but
+the rest of this makefile probably will not work.
+endef
+
+
+define defaultroutemsg
+
+The default route goes through the same network interface
+as the subnet you have specified to use for boot ($(DEFROUTE)).
+Running this as specified will break your network. Make sure
+you are using a different subnet for the boot provider than
+the one you use to connect to the internet
+endef
+
+define badroutemsg
+
+  The route for the specified network iface $(IFACE) does not map to $(DHCP_SUBNET_STUB).0,
+  but $(IFACEROUTE). This mismatch will make dhcpd fail.
+endef
+
+define setupmsg
+
+The current boot server configuration is
+  - Looking for a verdin at mac $(VERDIN_MAC)
+  - Providing DHCP, bootp, and nfs on network $(DHCP_SUBNET_STUB).0/224 which
+    is via interface $(SPECROUTE)
+  - Container will have the name $(CONTAINER_NAME)
+  - Serving from $(WORKDIR)/provide
+endef
+
+define noimgmsg
+
+No content is present to be served by NFS. Either extract an OE image such that its
+rootfs tarball is in $(WORKDIR)/provide/root and its bootfs tarball is in
+$(WORKDIR)/provide/boot, or run make again while providing an IMAGE=/path/to/image.tar
+definition
+endef
+
+define hostservicesmsg
+
+The system is currently running one or more nfs daemon processes (ps -e | grep nfsd has
+results) or sunrpc registration hosts (ps -e | grep rpc has results). This can happen
+if the container previously failed, if you have nfs-kernel-server installed and running
+on the host, or for some other mysterious reason. Unfortunately, the mechanisms of nfs
+mean that you can really only run nfs on either the container or the host, and trying on
+both makes a bunch of stuff in the container silently fail. You should remove
+nfs-kernel-server, or stop and mask rpcd.bind and nfs and everything involved in host
+systemd; and if you already did that and are getting this message, there are some left
+over processes that you need to kill -9 until nothing named nfsd is running on the host.
+endef
+
+.PHONY: config
+config: \
+  $(WORKDIR)/config/dhcpd.conf\
+  $(WORKDIR)/config/exports\
+  $(WORKDIR)/config/tftpd-hpa
+
+.PHONY: docker-build
+docker-build:\
+  $(WORKDIR)/config/exports \
+  $(WORKDIR)/config/dhcpd.conf \
+  $(WORKDIR)/config/tftpd-hpa \
+  $(WORKDIR)/provide/boot $(WORKDIR)/provide/root \
```

Comment:
make seems to be ignoring these prereqs, i was seeing this
`bash: line 0: cd: work/provide/boot: No such file or directory`

Reply:
guess it was just paths like these
cd $(WORKDIR)/provide/boot
testing it out with absolute paths instread